### PR TITLE
chore: custom dimension e2e

### PR DESCRIPTION
**Pull request by rephus**
Closes: #7585

### Description:

Testing custom dimensions on all warehouses 

Testing creating a custom dimension from UI (bin number only) 

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by rephus** [comment]
I'll create another PR when the bin_width is merged

File: packages/backend/src/database/seeds/development/02_saved_queries.ts [hunk 1]
```diff
@@ -466,4 +467,78 @@ export async function seed(knex: Knex): Promise<void> {
...
+                additionalMetrics: [],
+                customDimensions: [
+                    {
+                        id: 'amount_range',
+                        name: 'amount range',
+                        dimensionId: 'payments_amount',
+                        binType: BinType.FIXED_NUMBER,
+                        binNumber: 5,
+                        table: 'payments',
+                    } /* {
```
Comment (rephus) [inline packages/backend/src/database/seeds/development/02_saved_queries.ts:497]: I'll create another PR  with fixed_width to enable this when it gets merged

File: packages/e2e/cypress/e2e/app/customDimensions.cy.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    cy.contains('How do payment methods vary').click();
+    cy.contains('0-6');
+    cy.contains('6-12');
+};
+describe('Custom dimensions', () => {
+    beforeEach(() => {
+        cy.login();
+    });
+
+    it.only('I can create a custom dimension chart from api', () => {
```
Comment (ZeRego) [inline packages/e2e/cypress/e2e/app/customDimensions.cy.ts:92]: forgotten `.only` ?

Comment (rephus) [reply packages/e2e/cypress/e2e/app/customDimensions.cy.ts]: I initially added this to `test` the method locally, without having to createproject, then I forgot to add .skip . I changed this, what do you think ?

**Review by ZeRego — APPROVED** [review]

**Comment by rephus** [comment]
Snowflake test is failing becuase we haven't merged the rounding fix 
image

**Pull request merged**